pkgs/by-name/rd/rdio-scanner/package.nix

```
@@ -0,0 +1,47 @@
+{ lib
+, fetchFromGitHub
+, buildNpmPackage
+, buildGoModule
+}:
+
+let
+  version = "6.6.3";
+  src = fetchFromGitHub {
+    owner = "chuot";
+    repo = "rdio-scanner";
+    rev = "v${version}";
+    sha256 = "sha256-Icy5DQtjbpejyUCPGGTc5X+zrMccmUk9C7FNaG0H42U=";
+  };
+  rdio-scanner-client = buildNpmPackage {
+    pname = "rdio-scanner-client";
+    inherit version src;
+    patches = [ ./angular-outpath.patch ];
+    sourceRoot = "${src.name}/client";
+    npmDepsHash = "sha256-BmW4MUWhZflzBhsBqI2X52dtMtCVm+LgzTI7J8/B3OU=";
+    postInstall = ''
+      mkdir -p $out
+      cp -r /build/webapp $out
+    '';
+  };
+in
+buildGoModule {
+  pname = "rdio-scanner-server";
+  inherit version src;
+  sourceRoot = "${src.name}/server";
+  vendorHash = "sha256-Dvb8g+XtMcI9bbB83AZ94UI54L10jmBnXYrgzGe9200=";
+  postUnpack = ''
+    cp -r ${rdio-scanner-client}/webapp .
+  '';
+  postInstall = ''
+    mv $out/bin/server $out/bin/rdio-scanner
+  '';
+  meta = {
+    description = "The perfect software-defined radio companion.";
```

```suggestion
    description = "Perfect software-defined radio companion";
```
see contributing guide about meta.description